### ISSUE TITLE
Sankey / Flow Diagram

### ISSUE THREAD

**Issue by Butcherluke** [mostaphaRoudsari/EmergeSettlement#33]
@mostaphaRoudsari I know you mentioned at the weekend adding a Sankey diagram to the page. Something like would be a really powerful tool for linking it back to e[m]ergy as it could show the flows of energy much like an e[m]ergy diagram does. It could be on a separate tap though from the map?

**Comment by mostaphaRoudsari** [comment]
I agree that it will help. [This](http://bl.ocks.org/Neilos/584b9a5d44d5fe00f779) is a great example of bi-directional Sankey diagram. You can also double click on each component and see the underlying layer.
To do so I need the list of the component and a clean break down of numbers which goes between the sources. Do we have it already?

Also I think at some point we may need to hide some of the sliders and let it to user to expand them as a dropdown so we can add this diagram and keep the map big enough.

**Comment by jmskurtz** [comment]
i agree, simplification of the sliders will help free up more space. Been
thinking they can be on a single 1/4 page column on the left (or right)
that is either expandable or scrollable.
http://pixelscommander.com/polygon/htmlgl/demo/filters.html#.VR6vUPnF_pV

Then we could have the instructions in the second 1/4 page sidebar (Fixed)
and then an image in the other half of the page. the image can either be
the map or maybe you can choose from a drop down or slider to switch it to
the sanky, or emergy diagram.

I don't have adobe CS on this machine, but attached is a mock up...

On Thu, Apr 2, 2015 at 10:46 PM, Mostapha Sadeghipour Roudsari <
notifications@github.com> wrote:

> I agree that it will help. This
> http://bl.ocks.org/Neilos/584b9a5d44d5fe00f779 is a great example of
> bi-directional Sankey diagram. You can also double click on each component
> and see the underlying layer.
> To do so I need the list of the component and a clean break down of
> numbers which goes between the sources. Do we have it already?
> 
> Also I think at some point we may need to hide some of the sliders and let
> it to user to expand them as a dropdown so we can add this diagram and keep
> the map big enough.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/mostaphaRoudsari/SettlementEmerge/issues/33#issuecomment-89134430
> .

## 

_J I L L  M.  S O R N S O N   K U R T Z_
consultant http://www.revisionarch.com/  |  faculty
http://www.philau.edu/architectureandthebuiltenvironment/  |  fellow
http://www.design.upenn.edu/architecture/graduate/research/penn-tsinghua-tc-chan-center
  |  supporter http://www.rebuildsouthsudan.org/

**Comment by mostaphaRoudsari** [comment]
I think it is good to have some of the main sliders exposed and put the rest in collapse(able) menus. In that way we can teach people about the game step by step. Think about it as level 1 and level 2! Can you identify the critical ones and let me know?
